packages/test/test-service-load/src/utils.ts

```diff
@@ -58,6 +58,7 @@ class FileLogger extends TelemetryLogger implements ITelemetryBufferedLogger {
         this.baseLogger?.send(event);
```

Comment:
```suggestion
        this.baseLogger?.send({...event, hostName: pkgName});
```

Reply:
we don't need this to log in the file, as it's we know the host, as only stress logs to the file